### DISCUSSION TITLE
NeoOffice

### DISCUSSION THREAD

**Discussion by alb576** [elrumo/macOS_Big_Sur_icons_replacements#1442] [Icon request]
Libre Office and OpenOffice all have their nice Ventura-style icon, but NeoOffice not yet. I would really appreciate if someone could provide one for the community.
Thanks!

Original icon
 
![ship](https://user-images.githubusercontent.com/125752256/219874698-38ee9413-41e6-4ef0-b89a-d9abd6a6b19d.png)

**Comment by vitor-hugo** [comment]
Waiting for review...
https://macosicons.com/#/u/vitor.hugo

**Reply by alb576** [reply]
Just saw the icon! It's perfect. Thanks a lot!
